### ISSUE TITLE
IndexError

### ISSUE THREAD

**Issue by bitman** [Komodo/KomodoEdit#64]
When down-arrowing or clicking at end of the last line of an HTML file I get a notification alert saying "IndexError: list index out of range" with the only option being to "Disable Notifications of this Type". Is this expected behavior?

**Comment by Naatan** [comment]
Definitely not expected/desired behaviour. I am unable to reproduce the issue though, could you provide a sample snippet that you can reproduce the issue with (the shorter the better).

**Comment by bitman** [comment]
```
<!doctype html>
<html>
</html>
```

If I remove both html tags then click off the end then back on the end of the line again I get 
```
CodeIntelError:style_at_pos binary search sentinel hit: there is likely to be a logic problem here!
```

**Comment by Naatan** [comment]
Thanks @bitman, reproduced the issue for me.

**Comment by mitchell-as** [comment]
I cannot reproduce this. @Naatan what steps did you use? I did this:

1. Copy-pasted the snippet.
2. Saved to a new html file.
3. Removed the "html" start and end tags.
4. Moved my caret to the beginning of the file.
5. Moved my caret to the end of the file.
6. No popup and no problem.

**Comment by Naatan** [comment]
Reproduced using the exact steps you gave mitchell, interestingly I am unable to reproduce this with a clean profile :\ ugh, will try resetting parts of my profile to figure this out.

**Comment by Gedweb** [comment]
Have too when indexing php-project (Symfony2 framework)
IndexError: list index out of range

**Comment by Naatan** [comment]
This is resolved in 9.3.0
